icu4c/source/common/uprops.cpp

```diff
@@ -591,6 +591,10 @@ static int32_t scriptGetMaxValue(const IntProperty &/*prop*/, UProperty /*which*
 /*
  * Map some of the Grapheme Cluster Break values to Hangul Syllable Types.
  * Hangul_Syllable_Type is fully redundant with a subset of Grapheme_Cluster_Break.
+ *
+ * Starting with Unicode 16, this is not quite true:
+ * Some Kirat Rai vowels are given GCB=V for proper grapheme clustering, but
+ * they are of course not related to Hangul syllables.
```

Comment:
Suggest striking fully redundant if it's no longer true. Don't just append to the comment.

Reply:
ok done